hookrun/runhooks.go

```diff
@@ -155,8 +155,11 @@ func getScripts(request datastructures.HookRunRequest, workdir string) error {
 			return errors.Wrap(err, "Error retrieving hook script")
 		}
 		if resp.StatusCode != 200 {
-			fmt.Fprintln(os.Stderr, "Unable to retrieve hook script")
-			return errors.New("Unable to retrieve hook script")
+			msg := fmt.Sprintf(
+				"Unable to retrieve hook script, received status code %d",
+				resp.StatusCode)
+			fmt.Fprintln(os.Stderr, msg)
```

Comment:
Is there a reason you’re not using fmt.Fprintf?

Reply:
@puiterwijk yeah - I wanted to format the message one time and then use it in two places (the return statement below this line being the second).

Reply:
Ah, right! Thanks!